localization/english/event_localization/wc_scarlet_events_l_english.yml

```
@@ -0,0 +1,72 @@
+ SCARL.006_OPT_A:0 "The Saviours are Here!"
+
+ SCARL.007_TITLE:1 "Wrath of the Scarlets"
+ SCARL.007_DESC:0 "How many more of our brethren and our kind die or sacrificed before the Scourge is gone for good?\n\nHow long are we supposed to only dream about those peaceful times gone by?\n\nWe must take our fate in our own hands. We must cut the head of the snake. We must go to the darkest and icy depths of Northrend to slay the [GetTrait('lich_king').GetName(GetNullCharacter)] itself."
+ SCARL.007_OPT_A:0 "Send the word out for volunteers..."
+ SCARL.007_OPT_A_tt:0 "You send a message of your expedition to fellow Scarlets"
+
+ SCARL.008_TITLE:1 "I want YOU for the Scarlet Army"
+ SCARL.008_DESC:0 "To the brave warriors loyal to the scarlet cause,\nHow has the progress against removing the scourge from [GetTitleByKey('k_lordaeron').GetName] going? Not very well I presume?\nThey are numerous and unending in all the senses... aren't they?\nJoin me. Join my expedition to slay the [GetTrait('lich_king').GetName(GetNullCharacter)]! Our only hope in winning for Humanity lies in slaying the [GetTrait('lich_king').GetName(GetNullCharacter)]!"
+ SCARL.008_OPT_A:0 "This is madness!"
+ SCARL.008_OPT_B:0 "You are right... this is the only solution!"
+
+ SCARL.009_TITLE:1 "Foolish Endeavour"
+ SCARL.009_DESC:0 "To [scarlet_onslaught_leader.GetName],\nYou are either a fool or a bumbling idiot.\nGoing to the most icy hells where only wild creatures and the Scourge thrive? Where even the Light cannot give you enough warmth and hope?\nI wish you the best on your #italic "brave expedition" #!. I hope the Light blesses you nonetheless for your bravery.\nDo not concern us again."
+ SCARL.009_OPT_A:0 "Coward!"
+
+ SCARL.010_TITLE:1 "Brave Warrior"
+ SCARL.010_DESC:0 "To [scarlet_onslaught_leader.GetName],\nYou spoke my mind! The skirmishes across [GetTitleByKey('k_lordaeron').GetName] have, by far, done little to no progress...\nWe saw no other choice but to flee the kingdom as well but we are now ashamed of our behaviour!\nWe will atone for even letting such sinful thoughts flow inside us. We will join your expedition and crush the Scourge with our combined strenghts!"
+ SCARL.010_OPT_A:0 "Excellent..."
+
+ SCARL.011_TITLE:1 "Beginning of the End"
+ SCARL.011_DESC:0 "We have finally landed in [GetTitleByKey('c_moaki').GetName]. Very close to the [GetTrait('lich_king').GetName(GetNullCharacter)]'s lair but not within their grasp.\n\nFrom here we can launch our attacks on the Scourge and maybe even catch them by surprise?\n\nBut first, we will need to rid our new settlement from these... #italic animals #!that surround our new colony."
```

```suggestion
 SCARL.011_DESC:0 "We have finally landed in the [GetTitleByKey('c_moaki').GetName], within striking distance of the [GetTrait('lich_king').GetName(GetNullCharacter)]'s lair but not within its grasp.\n\nFrom here we can launch our attacks on the Scourge and, with [ROOT.Char.GetFaith.HighGodName]'s blessing, even catch them by surprise!\n\nBut first, we will need to rid our new colony of these... #italic animals#!... that surround our settlement."
```
keeping the pronoun consistent with "its"